crates/bevy_winit/src/system.rs

```diff
@@ -281,10 +283,12 @@ pub(crate) fn despawn_windows(
 }
 
 /// The cached state of the window so we can check which properties were changed from within the app.
-#[derive(Debug, Clone, Component)]
-pub struct CachedWindow {
-    pub window: Window,
-}
+#[derive(Debug, Clone, Component, Deref, DerefMut)]
+pub(crate) struct CachedWindow(Window);
```

Comment:
Making this `pub(crate)` is not breaking, it was already de facto on that visibility.